### PR TITLE
you can now deconstruct and build suit storage units!

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

adds the ability to both build (1x board, 2x manipulators) and deconstruct suit storage units. adds suit storage unit boards to the techweb, unlocked with the Industrial Engineering node.

## Why It's Good For The Game

more roboticists are building modsuits and shit like that, would be nice to give them the ability to organize them into actual storage units. + basic qol

## Changelog

:cl:
expansion: ability to print, construct, and deconstruct suit storage units
/:cl:

### CODE REVIEW

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 1]
```diff
@@ -824,6 +824,13 @@
 
 //Science
 
+/obj/item/circuitboard/machine/suit_storage
+	name = "Suit Storage Unit (Machine Board)"
+	greyscale_colors = CIRCUIT_COLOR_SCIENCE
```
Comment (Fikou) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm:829]: wouldnt it be engineering

Comment (magatsuchi) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm:829]: ah shit forgot i left it at science, thanks for catching it

**Comment by Jolly-66** [comment]
Didn't someone try to do this before? I could've sworn it got approved or what not, might be thinking of something else possibly.

**Comment by magatsuchi** [comment]
> Didn't someone try to do this before? I could've sworn it got approved or what not, might be thinking of something else possibly.

i have not a clue

**Comment by MrDoomBringer** [comment]
https://github.com/tgstation/tgstation/pull/55291

**Comment by magatsuchi** [comment]
@MrDoomBringer that one was closed sheerly by the author's will, so i'm not too worried about a maintainer closing this one over that

File: code/game/machinery/suit_storage_unit.dm [hunk 2]
```diff
@@ -211,13 +212,6 @@
 	storage = null
 	set_occupant(null)
 
-/obj/machinery/suit_storage_unit/deconstruct(disassembled = TRUE)
```
Comment (OrionTheFox) [inline code/game/machinery/suit_storage_unit.dm:214]: shouldnt this stay so that it drops everything on deconstruction still? just change the drop/qdel?

**Comment by Donpedrito** [comment]
This is a bit random, I know, but consider checking if this fixes #60040; I imagine it might, since zombies are able to attack other machines.

File: code/game/machinery/suit_storage_unit.dm (hunk 2 above)
Comment (magatsuchi) [reply code/game/machinery/suit_storage_unit.dm:214]: will look at this, busy atm

**Comment by VladinXXV** [comment]
I think it'd make more sense to use lasers and/or capacitors. (because of the UV decontamination bit.) It might be worth looking into having better parts do something, too. Like... faster decontamination. Or modsuit recharging. Do they do that already? If they do they can do it faster. If not they can gain the ability.

**Pull request closed without merging**